### ISSUE TITLE
infinite loop loading when using loader divs with v-if

### ISSUE THREAD

**Issue by nkwinder** [ratiw/vuetable-2#405]
I don't know if i'm doing something wrong, or this is a vue bug, or a vuetable bug.

https://codepen.io/anon/pen/EEXewo

I just added some divs that are shown when vuetable is loading. When using v-if, there is an infinite loop (when completed loading, vuetable starts loading again). If  v-show is used, instead of v-if in these divs... then there is no problem.

thanks

**Comment by ratiw** [comment]
@nkwinder It's a bit strange. Try changing `v-if` to `v-show` instead.

**Comment by nkwinder** [comment]
Yes as i said v-show is working. Changing the div order also works... (moving the overlay div before vuetable). 
But this is very strange behavior and i would like to know whats the problem here.

**Comment by ratiw** [comment]
I don't know for sure what caused it, but I guess that `v-if` might have forced re-evaluation of the component as it reinsert the dom. This is not happening with `v-show` as it only show/hide the dom.
